### ISSUE TITLE
Feature Request: Useless quest items

### ISSUE THREAD

**Issue by S-Flavius** [mcgeer/WastedBankSpace#38]
It would be nice if the plugin could show you a list of all the trash items that can be destroyed and have no use in any future quests

**Comment by JonStudders** [comment]
I do not believe this is worth the amount of code required when it's as simple as just going to Draynor.
We would have to, create a new list of items usable for only quests, check if quests have been done then flag it.
This will increase processing time, we are attempting to keep an O(1) time for lookups.
@mcgeer do you agree?

**Comment by mcgeer** [comment]
Same sentiment. A low priority change can be made to have a clickable button in the rl panel that runs a more complex check and returns details such as this. 
But idly doing these checks is a waste imo.
